.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+# Contributing to cs102
+
+Keep in mind that this is a first year CS course. 
+
+Also keep in mind that there are categories of content:
+* Topics
+  * Asides
+* Code
+  * Tests
+* Assignments
+* Labs
+* Extras
+  * Course main page
+  * Course Config
+  * Outline
+  * Getting Started
+  * .md files
+  * IDE setup
+  * Gradle stuff
+
+## How to Contribute
+### Issues
+* Use issues to point out bugs/issues
+* Use issues to suggest enhancements
+* Use issues to suggest content
+* Use issues to suggest changes
+
+#### Submitting a Good Issue
+* Use the template
+
+### Pull Requests
+* Use the template
```

Comment:
Templates are automatically used. Instead of "use the template" I'd expand on this to effectively say "try to communicate effectively." Loads of issues/PRs that "use the template" are just the template text, totally unchanged.